src/components/05_pages/NodeForm/index.js

```diff
@@ -189,7 +189,12 @@ class NodeForm extends React.Component {
                         onChange: (propType === 'attributes'
                           ? this.onAttributeChange
                           : this.onRelationshipChange)(fieldName),
-                        inputProps,
+                        inputProps: {
+                          ...inputProps,
+                          required: this.props.schema.properties.data.properties.attributes.required.includes(
```

Comment:
I'm not sure if adding this to input props is a good approach. It seems like this could have a broader effect than just how the input is rendered. Could we just pass a prop required?

Reply:
I think I do agree with lauriii, required seems such a generic thing that it should live on the component itself. Note: We also don't check the propType yet, so required relationships aren't taken into account

Reply:
I had only attached this to `inputProps` since it seemed sensible since those props should be passed to the `input` element.

Reply:
Don't we actually want to set it on the form control as well, so the label indicates the requiriedness?